app/components/hiring_staff/vacancy_form_page_heading_component.rb

```diff
@@ -0,0 +1,52 @@
+class HiringStaff::VacancyFormPageHeadingComponent < ViewComponent::Base
+  delegate :current_organisation, to: :helpers
+  delegate :current_step, to: :helpers
+  delegate :total_steps, to: :helpers
+
+  def initialize(vacancy, session_vacancy_attributes)
```

Comment:
also how we pass arguments to the component - probably out of scope of the ticket, but something for us to think about

Reply:
Open to other ideas about how to pass arguments in. Originally I had two separate arguments for the two parts of the session[:vacancy_attributes] that are actually used. That resulted in rather ugly views like

```ruby
= render HiringStaff:: VacancyFormPageHeadingComponent.new(@vacancy, session[:vacancy_attributes]&.[]('job_location'), session[:vacancy_attributes]&.[]('readable_job_location'))
```